.github/workflows/jaas-smoke.yml

```
@@ -0,0 +1,72 @@
+name: "Juju JAAS Smoke Test"
+on:
+  pull_request:
```

Actually, I think to be consistent with the other jobs, lets go for this incantation.
```suggestion
on:
  push:
    branches: ["[0-9].[0-9]+", "[0-9].[0-9]+.[0-9]+", main]
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
  workflow_dispatch:
```